components/meter/stories/meter.stories.js

```diff
@@ -33,12 +33,26 @@ export default {
 			options: ["info", "notice", "positive", "negative"],
 			control: "select",
 		},
+		helpText: {
```

Comment:
Did we want to add a help text story for meter? I did see an example in the token specs, but help text doesn't show up on the desktop designs. It's also not in the Figma properties, so maybe we don't need it (maybe that part of the token specs is unnecessary and could be removed?)

<img width="476" alt="Screenshot 2025-06-25 at 5 18 26 PM" src="https://github.com/user-attachments/assets/4a66852b-5d9d-4034-84aa-35e875263246" />

I feel like if we have a control and styles for it, we should probably have a story too.